pkg/db/db_local/start_services.go

```diff
@@ -59,18 +59,21 @@ const (
 	ListenTypeLocal = "local"
 )
 
-// IsValid is a validator for StartListenType known values
-func (slt StartListenType) IsValid() error {
+// ToListenAddresses is transforms StartListenType known aliases into their actual value
+func (slt StartListenType) ToListenAddresses() string {
 	switch slt {
-	case ListenTypeNetwork, ListenTypeLocal:
-		return nil
+	case ListenTypeNetwork:
+		return "*"
```

Comment:
We probably need to also convert `0.0.0.0` into `*`, or specify that `0.0.0.0` cannot be passed in the `listen` database option and that `network` or `*` must be used in this case.